### PR TITLE
New: Added correctness classes to progress bars and arialabels to drawer buttons

### LINKED ISSUE #224: Add correctness class to progress bars
### Subject of the issue
Will facilitate differentiation on display of correctness alongside completion

### PR DESCRIPTION
**Pull request by oliverfoster**
fixes #224 

### Fix
* Added correctness classes to progress bars
* Added correctness arialabels to drawer buttons

### CODE REVIEW

File: js/PageLevelProgressIndicatorView.js [hunk 1]
```diff
@@ -76,10 +76,14 @@ class PageLevelProgressIndicatorView extends Backbone.View {
   }
 
   checkCompletion() {
-    const percentage = this.setPercentageComplete();
```
Comment (cahirodoherty-learningpool) [inline js/PageLevelProgressIndicatorView.js:79]: Minor: I think the `return percentage` can be dropped from the setPercentageComplete function now as it isn't used anywhere

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Comment by oliverfoster** [comment]
> This works as expected thanks. I just have a couple of queries...
> 
> 1. Should `.is-partially-correct` class be included for consistency with [Tutor classes](https://github.com/adaptlearning/adapt-contrib-tutor/blob/ac27412e5ed8c6f47b301dab8a471ae77f069e9d/js/TutorModel.js#L45).
> 2. Should the [`aria-label`](https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/blob/e52bf68b2b5fc11a8d07c0003beb94294ee5a975/templates/pageLevelProgressItem.jsx#L55) include correctness labels too?

I notice that we don't have an aria label for partly correct. Do we need to add one? It would make sense to me.

https://github.com/adaptlearning/adapt_framework/blob/917b16180a57a8ed3c492c99da4531910fb62660/src/course/en/course.json#L46-L47
https://github.com/adaptlearning/adapt_framework/blob/917b16180a57a8ed3c492c99da4531910fb62660/src/course/en/course.json#L62-L63

**Review by kirsty-hames — APPROVED** [review]
Works as expected thanks 👍

**Comment by kirsty-hames** [comment]
> > This works as expected thanks. I just have a couple of queries...
> > 
> > 1. Should `.is-partially-correct` class be included for consistency with [Tutor classes](https://github.com/adaptlearning/adapt-contrib-tutor/blob/ac27412e5ed8c6f47b301dab8a471ae77f069e9d/js/TutorModel.js#L45).
> > 2. Should the [`aria-label`](https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/blob/e52bf68b2b5fc11a8d07c0003beb94294ee5a975/templates/pageLevelProgressItem.jsx#L55) include correctness labels too?
> 
> I notice that we don't have an aria label for partly correct. Do we need to add one? It would make sense to me.
> 
> #### course.json
> https://github.com/adaptlearning/adapt_framework/blob/917b16180a57a8ed3c492c99da4531910fb62660/src/course/en/course.json#L46-L47 https://github.com/adaptlearning/adapt_framework/blob/917b16180a57a8ed3c492c99da4531910fb62660/src/course/en/course.json#L62-L63
> 
> #### buttonsView.js
> https://github.com/adaptlearning/adapt-contrib-core/blob/4b8a99ab60704bb6d6993992ceae575b86a120c7/js/views/buttonsView.js#L188-L194

Agreed. Issue raised https://github.com/adaptlearning/adapt_framework/issues/3565

**Comment by oliverfoster** [comment]
I haven't included the arialabels for the plp drawer items yet. I'll wait for the issue mentioned to be resolved.

**Comment by kirsty-hames** [comment]
> adaptlearning/adapt_framework#3565



> I haven't included the arialabels for the plp drawer items yet. I'll wait for the issue mentioned to be resolved.

PR ready for review https://github.com/adaptlearning/adapt_framework/pull/3570

**Comment by oliverfoster** [comment]
This has the aria labels in it now, and support for `_canShowMarking`. Could you re-review please? @kirsty-hames

**Review by kirsty-hames — APPROVED** [review]
This works as expected thanks 👍

**Pull request merged**